packages/components/src/Input.js

```diff
@@ -1,26 +1,52 @@
 import React from 'react'
 import Box from './Box'
 
-export const Input = React.forwardRef(function Input(props, ref) {
+import { get } from '@theme-ui/css'
+
+/** @type {import('theme-ui').ThemeUIStyleObject} */
+const autofillStyles = {
+  boxShadow: 'inset 0 0 0 1000px var(--theme-ui-input-autofill-bg)',
+  fontSize: 'inherit',
+  ':first-line': {
+    fontSize: '1rem',
+  },
+}
+
+/** @type {import('theme-ui').ThemeUIStyleObject} */
+const defaultInputStyles = {
+  display: 'block',
+  width: '100%',
+  p: 2,
+  appearance: 'none',
+  fontSize: 'inherit',
+  lineHeight: 'inherit',
+  border: '1px solid',
+  borderRadius: 4,
+  color: 'inherit',
+  bg: 'transparent',
+
+  ':autofill, :autofill:hover, :autofill:focus': autofillStyles,
+  ':-webkit-autofill, :-webkit-autofill:hover, :-webkit-autofill:focus':
+    autofillStyles,
+}
+
+export const Input = React.forwardRef(function Input(
+  { sx, autofillBackgroundColor = 'background', ...rest },
```

Comment:
Should we use `autofillBgColor` to be more consistent? Idk

Reply:
Actually I don't think we support `bgColor` anywhere. It's either `bg` or `backgroundColor` or `background`.